README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is in `stripe` using `ACH`.
+When capturing a bank account the flow can be divided in two steps:
+
+- Capturing
+- Verification
+
+Once the `Capture` has been made, `stripe` will deposit two random amounts on the bank account. These amounts have to be
+verified against (or in other words, *sent to*) `stripe`. You can do the verification process directly using `billforward-js`.
+
+In more details what happens is:
+- Capture a bank account using `billforward-js`
```

Comment:
Nitpick, but:

- `billforward-js` is the name of the repository.
- BillForward.js is the proper name of the library.
- BFJS is acceptable in an informal context, but this readme is more formal.

I prefer BillForward.js here. Mainly because inline code formatting is clearest when it is _only_ used to refer to inline code.